src/main/kotlin/xyz/wagyourtail/unimined/UniminedExtensionImpl.kt

```diff
@@ -310,7 +347,7 @@ open class UniminedExtensionImpl(project: Project) : UniminedExtension(project)
 
     private fun afterEvaluate() {
         for ((sourceSet, mc) in minecrafts) {
-            (mc as MinecraftProvider).afterEvaluate()
+            if (mc is MinecraftProvider) mc.afterEvaluate()
```

Comment:
This might not be necessary, it should work as it was, now. But in the future if we want to implement MinecraftConfig without overriding MinecraftProvider it might become a necessity.

So this can probably be reverted for now.